charts/nitro/templates/statefulset.yaml

```
@@ -60,8 +60,17 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           {{- if .Values.diagnosticMode }}
-          command: ["sleep"]
-          args: ["infinity"]
+          command: ["/bin/sh"]
+          args:
+            - -c
+            - |
+              #!/usr/bin/env bash
+              set -eo pipefail
```

I got the following error when running this on a test pod `/bin/sh: 2: set: Illegal option -o pipefail`
```suggestion
              set -e pipefail
```